packages/classic/scss/spreadsheet/_variables.scss

```diff
@@ -2,7 +2,7 @@
 $kendo-spreadsheet-border-width: 1px !default;
 $kendo-spreadsheet-font-family: $kendo-font-family !default;
 $kendo-spreadsheet-font-size: $kendo-font-size-md !default;
-$kendo-spreadsheet-line-height: $kendo-line-height-md !default;
+$kendo-spreadsheet-line-height: 20px !default;
```

Comment:
do we need to change this value? As I see in the themes the $kendo-line-height-md = $kendo-line-height = k-math-div( 20, 14 ), which is the same